testsuite/pytest_extensions.py

```diff
@@ -137,3 +161,146 @@ def test_CIMClass_equal(
 
             ret = None  # Debugging hint
     return ret
+
+
+def test_function_new(test_func):
```

Comment:
Since this is probably what will be the major test function decorator in the future, shouldn't we call it something else that _new.   Expanded, complete, 2. Not sure of name but new is just transitional as a name.

Reply:
We agreed to remove the old function and to name zhe new function `simplified_test_function()`.

Reply:
DONE